src/pages/Login/Login.scss

```
@@ -1 +1,90 @@
-s
\ No newline at end of file
+.loginInputCommon {
+  display: block;
+  margin: 0 auto;
+  width: 80%;
+}
+
+@mixin loginInput {
+  @extend .loginInputCommon;
+  padding: 6px 10px;
+  height: 45px;
+  border: 1px solid grey;
+}
+
+@mixin loginButton {
+  @extend .loginInputCommon;
+  height: 40px;
+}
```

```suggestion
%loginInputCommon {
  display: block;
  margin: 0 auto;
  width: 80%;
}

~~~~

.login {
  .loginInput {
    @extend %loginInputCommon;
     padding: 6px 10px;
     height: 45px;
     border: 1px solid grey;
   }
   
   .loginId {
      @extends .loginInput
      margin-bottom:4px;
   }
}
```

대략 이런식으로 사용하시면 됩니다! 반드시 mixin을 거쳐서 사용해야될 필요는 없습니다!
mixin -> 자주 사용되는 속성들을 모아놓고 그걸 각 스타일부분에서 가져와서 쓰는 것
extends -> 기본이 되는 틀이 있고 그걸 확장해서 각자 다르게 조금씩 추가해서 쓰는 것
정도로 인식하시면 됩니다!

%로 쓰는것과 실제 className으로 쓰는것의 차이는 css로 컴파일 되는지 안되는지입니다. 
이부분은 한번 검색해서 참고해보시면 좋을 것 같네요!

---

수정했습니다~!